cmd/podman/containers/create.go

```diff
@@ -324,3 +339,20 @@ func createPodIfNecessary(s *specgen.SpecGenerator, netOpts *entities.NetOptions
 	s.Pod = podName
 	return registry.ContainerEngine().PodCreate(context.Background(), createOptions)
 }
+
+// inheritPodUserNS gets the userns option that the pod was created with and sets that
+// for the container being created (if the container is being created inside a pod).
+func inheritPodUserNS(s *specgen.SpecGenerator) error {
```

Comment:
I think this should happen on the server side.  If I launch a container within a pod via the API, I want this to happen also.